llvm/include/llvm/ProfileData/MemProfData.inc

```diff
@@ -229,6 +228,41 @@ void Merge(const MemInfoBlock &newMIB) {
 } __attribute__((__packed__));
 #endif
 
+constexpr int MantissaBits = 12;
+constexpr int ExponentBits = 4;
+constexpr uint16_t MaxMantissa = (1U << MantissaBits) - 1;
+constexpr uint16_t MaxExponent = (1U << ExponentBits) - 1;
+
+// Encodes a 64-bit unsigned integer into a 16-bit scaled integer format.
+inline uint16_t encodeHistogramCount(uint64_t Count) {
+  if (Count == 0)
+    return 0;
+
+  const uint64_t MaxRepresentableValue = static_cast<uint64_t>(MaxMantissa)
```

Comment:
Can this be moved out of the function and made a constexpr like the other Max values above?

Reply:
did this get updated?

Reply:
It did not get updated. I made the changes to MemProfData.inc in compiler-rt and promptly copied the contents of the version in ProfileData over to compiler-rt. Thanks for catching this.